RNF34.py

```diff
@@ -0,0 +1,134 @@
+from abc import ABC, abstractmethod
+
+
+class BiologicalSequence(ABC):
+    @abstractmethod
+    def __init__(self, sequence):
+        pass
+
+    @abstractmethod
+    def __len__(self):
+        pass
+
+    @abstractmethod
+    def __getitem__(self, slc):
+        pass
+
+    @abstractmethod
+    def __str__(self):
+        pass
+
+    @abstractmethod
+    def __repr__(self):
+        pass
+
+    @abstractmethod
+    def is_valid_alphabet(self):
+        pass
+
+
+class NucleicAcid(BiologicalSequence):
+    def __init__(self, sequence):
+        self.sequence = sequence
+
+    def __len__(self):
+        return len(self.sequence)
+
+    def __getitem__(self, slc):
+        return self.sequence[slc]
+
+    def __str__(self):
+        return str(self.sequence)
```

Comment:
Хорошо, что перестраховались и точно определили строковую переменную) Но, тут это не обязательно, так как self.sequence - уже строка.
```suggestion
        return self.sequence
```